### ISSUE TITLE
[a11y][macOS] Blank space from “Something went wrong..”  screen is narrated as a text element by the VoiceOver

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#8394]
**Important details:**

* not reproducing on Windows; 

**Affected versions:**

* 2.18.0 (2.202310241802)

**Tested Platforms:**

* macOS 12.6.6;

**Prerequisites:**

* Have Mozilla VPN app installed and open;
* Be signed out; 
* Be in “Something went wrong..”screen (this can be triggered by being in the Email/Password screen and running the ‘force_heartbeat_failure’ command in the [+Inspector+](https://mozilla-mobile.github.io/mozilla-vpn-client/inspector/)); 
* Have the VoiceOver enabled and quick nav as well; 

**Steps to reproduce:**

1. Navigate through the “Something went wrong..” screen while using the right arrow (macOS) until you reach the “Check outage updates” link;
1. Observe the behavior;

**Expected result:**

* The VoiceOver narrates only the visible elements from the screen;

**Actual result:**

* The VoiceOver narrates the blank space between the “Unable to establish..” text and “Check outage updates” link as a text element;

**Notes:**

* See the attached video;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5769)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on Mozilla VPN 2.20.0 (2.202312070143), using macOS 12.6.6. 

Blank space from “Something went wrong..” screen is no longer narrated as a text element by the VoiceOver.

Attaching postfix video.

!Screen Recording 2023-12-07 at 11.21.48.mov|width=3584,height=1888!

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on Mozilla VPN 2.20.0 (2.202312070143), using macOS 12.6.6. 

Blank space from “Something went wrong..” screen is no longer narrated as a text element by the VoiceOver.

Attaching postfix video.

!Screen Recording 2023-12-07 at 11.21.48 (c33a63c5-902b-4fff-a1e4-bf047084b4a5).mov|width=3584,height=1888!
